spyder/widgets/mixins.py

```diff
@@ -142,7 +144,7 @@ def _get_inspect_shortcut(self):
 
     def _format_text(self, title=None, signature=None, text=None,
                      inspect_word=None, title_color=None, max_lines=None,
-                     display_link=False):
+                     width=None, display_link=False):

```

Comment:
It makes no sense to make `width=None` because `textwrap.wrap` won't accept it as a parameter. So let's give it a default value of `60`.